dashboard/test/models/survey_result_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class SurveyResultTest < ActiveSupport::TestCase
+  test "proper SurveyResult members" do
+    assert SurveyResult::ETHNICITIES["asian"] == "Asian"
+    assert SurveyResult::RESULT_ATTRS.include? "survey2016_ethnicity_asian"
+  end
```

Comment:
hm, well, your model is not doing much so there is not much to test. Should it be doing anything else?